packages/docs/blog/2023-08-08-system-strings.md

```diff
@@ -0,0 +1,206 @@
+---
+slug: demystifying-fhir-system-strings
+title: Demystifying FHIR System Strings
+authors:
+  name: Rahul Agarwal
+  title: Medplum Core Team
+  url: https://github.com/rahul1
+  image_url: https://github.com/rahul1.png
+  tags: [fhir-datastore]
+---
+
+import CodeBlock from '@theme/CodeBlock'
+
+# Demystifying FHIR System Strings
+
+One of the main sources of confusion when starting out with FHIR are `system` strings.
```

Comment:
One of the main sources of confusion when starting an implementation is with FHIR are `system` strings.